charts/cni-metrics-helper/templates/deployment.yaml

```diff
@@ -5,7 +5,10 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     k8s-app: cni-metrics-helper
+{{ include "cni-metrics-helper.labels" . | indent 4 }}
```

Comment:
Without this change, command from [NOTES.txt](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/charts/cni-metrics-helper/templates/NOTES.txt) would not work.

Reply:
This will add label to the deployment, which is fine.

I think, there is an additional change required in order to meet the NOTES.txt description.

```
diff --git a/charts/cni-metrics-helper/templates/deployment.yaml b/charts/cni-metrics-helper/templates/deployment.yaml
index 5cb2fb13..a33ccc44 100644
--- a/charts/cni-metrics-helper/templates/deployment.yaml
+++ b/charts/cni-metrics-helper/templates/deployment.yaml
@@ -23,6 +23,8 @@ spec:
       {{- end }}
       {{- end }}
       labels:
+        app.kubernetes.io/name: {{ include "cni-metrics-helper.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
         k8s-app: cni-metrics-helper
     spec:
       containers:
```

You can bring this in separately.

Reply:
Oops... my mistake, thanks for fixing that.